go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	go.uber.org/zap v1.16.0
 	golang.org/x/mod v0.4.1 // indirect
 	golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f
-	golang.org/x/tools v0.0.0-20210114065538-d78b04bdf963 // indirect
+	golang.org/x/tools v0.0.0-20210115202250-e0d201561e39 // indirect
```

Comment:
We do not need to change our go mod files here

Reply:
was done by `go mod vendor` . wouldn't affect anything imo